### PR TITLE
PLATFORM-852 Helios Login log passwords comparison results from Helios a...

### PR DESCRIPTION
**Pull request by ghost**
...nd MediaWiki

### CODE REVIEW

File: extensions/wikia/Helios/User.class.php [hunk 1]
```diff
@@ -123,7 +123,8 @@ public static function comparePasswordCheck( $bHeliosCheck, $bHelios, $bMediaWik
 			\Wikia\Logger\WikiaLogger::instance()->error(
 				'HELIOS_LOGIN',
 				[ 'method' => __METHOD__, 'type' => $sType, 'hash' => $sHash,
-				'user_id' => $iUserId, 'username' => $sUserName ]
+				'user_id' => $iUserId, 'username' => $sUserName,
+				'helios' => $bHelios, 'mediawiki' => $bMediaWiki ]
```
Comment (pchojnacki) [inline extensions/wikia/Helios/User.class.php]: This is just boolean right ?

**Comment by pchojnacki** [comment]
If boolean then LGTM

File: extensions/wikia/Helios/User.class.php (hunk 1 above)
Comment (Grunny) [inline extensions/wikia/Helios/User.class.php]: Yeah, both boolean only from a quick check.

**Comment by Grunny** [comment]
LGTM.

**Pull request merged**